src/utils/ansi-colors.ts

```
@@ -0,0 +1,106 @@
+      low: '🟢',
+      medium: '🟡',
+      high: '🔴',
+      critical: '🚨',
+      vip: '👑'
+    };
+    
+    return `${emojiMap[priority]} ${this.colorize(text, colorMap[priority])}`;
+  }
+
+  static stripAnsi(text: string): string {
+    return text.replace(/\x1B\[[0-9;]*m/g, '');
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)